src/components/v2/values/chartValuesDiff/ChartValuesView.scss

```diff
@@ -376,12 +391,12 @@
                     }
                 }
             }
+        }
```

Comment:
it seems extra. please verify

Reply:
Ohh no. I have moved `.chart-values-view__update-cta` outside hence it's showing like this.